core/internal/features_test.go

```diff
@@ -298,14 +296,12 @@ func TestIntegration_RunLog(t *testing.T) {
 			logBlockHash:     triggeringBlockHash,
 			receiptBlockHash: triggeringBlockHash,
 			wantStatus:       models.RunStatusCompleted,
-			wantFinishedAt:   true,
 		},
 		{
 			name:             "ommered request",
 			logBlockHash:     triggeringBlockHash,
 			receiptBlockHash: otherBlockHash,
 			wantStatus:       models.RunStatusErrored,
-			wantFinishedAt:   false,
```

Comment:
As discussed with @se3000, this should always be set when the job is "finished".